testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeRedirect.java

```diff
@@ -877,6 +877,30 @@ public void testViewReferencingHiveAndDeltaTable(boolean legacyHiveViewTranslati
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testHiveToDeltaPropertiesRedirect()
+    {
+        String tableName = "test_redirect_to_delta_properties_" + randomNameSuffix();
+
+        onDelta().executeQuery("" +
+                "CREATE TABLE " + tableName + " USING DELTA " +
+                "LOCATION '" + locationForTable(tableName) + "' " +
+                " AS SELECT true AS a_boolean");
+
+        List<Row> expected = List.of(
+                row("delta.minReaderVersion", "1"),
+                row("delta.minWriterVersion", "2"));
+
+        try {
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.default.\"%s$properties\"", tableName))).containsOnly(expected);
+            assertThat(onTrino().executeQuery(format("SELECT * FROM hive.default.\"%s$properties\"", tableName))).containsOnly(expected);
+        }
+        finally {
+            onDelta().executeQuery("DROP TABLE default." + tableName);
```

Comment:
```suggestion
                  dropDeltaTableWithRetry("default." + tableName);
```

This one is needed because dropping on Databricks may encounter concurrent modification issues related to Databricks updating async the schema of the table on AWS Glue.